app/models/concerns/user_search.rb

```diff
@@ -7,7 +7,7 @@ module UserSearch
   included do
     # user: first_name, last_name, email, mobile
     scope :search, ->(options = {}) {
-      search_text = options[:search_text] || ''
+      search_text = options[:search_text].downcase || ''
```

Comment:
Reading this again, it's not actually needed as we are doing an `ILIKE` search in SQL which is a case-insensitive search.